prototyper/Cargo.toml

```
@@ -22,10 +22,14 @@ spin = "0.9.8"
 uart16550 = "0.0.1"
 riscv-decode = "0.2.1"
 fast-trap = { version = "0.0.1", features = ["riscv-m"] }
+uart_xilinx = { git = "https://github.com/duskmoon314/uart-rs/" }
 
 
 [[bin]]
 name = "rustsbi-prototyper"
 test = false
 bench = false
 
+[features]
+nemu = []
+fw_payload = []
```

```suggestion
payload = []
```

Will this name be better?